openshift/openshift/src/main/java/org/arquillian/cube/openshift/impl/enricher/AwaitRoute.java

```diff
@@ -0,0 +1,27 @@
+package org.arquillian.cube.openshift.impl.enricher;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+import java.util.concurrent.TimeUnit;
+
+import javax.inject.Qualifier;
+
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+/**
+ * Used inside {@link RouteURL} to configure awating the route URL to be available.
+ * Availability is defined as returning a known good HTTP status code.
+ */
+@Qualifier
+@Documented
+@Retention(RUNTIME)
+@Target({ElementType.FIELD, ElementType.PARAMETER})
+public @interface AwaitRoute {
```

Comment:
What's happening if there is nothing in default root location? I think an attribute like `context` should be added, which by default is set to `/`

Reply:
Ah totally makes sense!